app/views/main.html

```diff
@@ -32,11 +32,14 @@
   <option value="3">Slide 4</option>
 </select>
 
-<simple-slider style="width:612px; height:612px" auto-play="false" change="{{actualslide}}">
+<button ng-click="slider.prev()">Prev Slide</button>
+<button ng-click="slider.next()">Next Slide</button>
```

Comment:
Glad it was self explaining, if you want further uses let me know, I used it on a test for some job. 

Also added a demo with dynamic images :wink: